templates/_helpers.tpl

```diff
@@ -210,6 +210,23 @@ define this here as well to simplify comparison with packs-volume-mounts
   {{- end }}
 {{- end -}}
 
+#Inserted for override ability to happen via helm charts
+
+{{- define "stackstorm-ha.override-config-mounts" -}}
```

Comment:
Hmm. Could we rename both of these helpers to use "overrides" instead of "override" (to consistently use the feature's name)?
I know it's nitpicky. Sorry.